torq.q

```diff
@@ -611,16 +611,29 @@ if[@[value;`.ps.loaded;0b]; .ps.initialise[]]
 // initialise connections
 if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
+// function to execute functions in .proc.initlist
 .proc.try:{[id;a]
-  .lg.o[id;"attemping to run: ",.Q.s1 a];
-  @[value;a;{[id;a;x].lg.e[id;x," error - failed to run: ",.Q.s1 a]}[id;a]];
+  .lg.o[id;"attempting to run: ",.Q.s1 a];
+  success:@[value;a;{[id;a;x].lg.e[id;x," error - failed to run: ",.Q.s1 a];:1b}[id;a]];
+  if[-1h=type success;:()];
+  .proc.initexecuted,:a;
+  .lg.o[id;"run successful: ",.Q.s1 a];
+ }
+
+.proc.initcmd:{
+  // command line functions executed first
+  $[`initlist in key .proc.params;
```

Comment:
If think you should change this to error out, like `.proc.init` was before your changes:
```
if[not`initlist in key .proc.params;:.lg.o ... ];
{.proc.try[`init;(x;`)]}each .proc.params`initlist;
```